vrx_gazebo/src/wildlife_scoring_plugin.cc

```diff
@@ -461,12 +461,14 @@ void WildlifeScoringPlugin::PublishAnimalLocations()
     // Conversion from Gazebo Cartesian coordinates to spherical.
 #if GAZEBO_MAJOR_VERSION >= 8
     const ignition::math::Pose3d pose = buoy.link->WorldPose();
+    const ignition::math::Vector3d position = this->world->SphericalCoords()->GlobalFromLocal(pose.Pos());
```

Comment:
Could you split this line into two lines? The linter complains because it's > 80 chars.

Reply:
No problem!